source/signaling_api.c

```diff
@@ -0,0 +1,1810 @@
+/* Standard includes. */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* API includes. */
+#include "signaling_api.h"
+
+/* CoreJSON includes. */
+#include "core_json.h"
+
+/**
+ * Helper macro to check if the AWS region is China region.
+ */
+#define SIGNALING_IS_CHINA_REGION( pAwsRegion )     \

```

Comment:
What about other regions? 

https://github.com/awslabs/amazon-kinesis-video-streams-producer-c/blob/3e519b7670e39031375d227f983ad2cde888078e/tst/ProducerCallbackProviderApiTest.cpp#L66-L88

Reply:
Thanks for sharing! I refer to [this implementation](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/4f1d3add4c0f88212fe6646898e5aba140ecbb3c/src/source/Signaling/ChannelInfo.c#L183). Could you help me understand if we need more special rule for other region? Is it necessary to apply that into WebRTC?

Reply:
This is correct we do not need special rule for other regions for webrtc.  If they are needed in the future they can be added as requested.

Reply:
I reached out to confirm the above, and what you mentioned is accurate .